### ISSUE TITLE
Building docs instructions

### ISSUE THREAD

**Issue by jayvdb** [coala/coala#5449]
The instructions on https://api.coala.io/en/latest/Developers/Development_Setup.html are wrong.

No need to change to docs repo.

**Comment by jayvdb** [comment]
Also need to explain that the package must be installed first, or set PYTHONPATH

**Comment by jayvdb** [comment]
See https://github.com/coala/coala/pull/5426 for syntax

**Comment by monsij** [comment]
May I know how can the changes be made to effect if no change is to be made to the repository?

**Comment by bekicot** [comment]
@jayvdb Is this task is available to take?

**Comment by jayvdb** [comment]
https://gitlab.com/coala/mobans/issues/8 should be done first

**Comment by bekicot** [comment]
@jayvdb https://gitlab.com/coala/mobans/issues/8 is done.
But the `setup.py` is not updated yet, is there any issue for that yet?

I want to finish https://github.com/coala/coala/pull/5470 this and seems like it will still be blocked as long as the `setup.py` is not updated to reflect `moban` changes.

**Comment by jayvdb** [comment]
correct. waiting for https://gitlab.com/coala/mobans/merge_requests/22 , and then will sync everything

**Comment by jayvdb** [comment]
This section of the docs should talk about both documentation and coala repos , and explain most coala repos now have generatef sphinx docs.

**Comment by sourabh1031** [comment]
@jayvdb please assign this to me

**Comment by li-boxuan** [comment]
@sourabh1031 Can you finish your open PR https://github.com/coala/coala/pull/5772 first?

**Comment by Anmolbansal1** [comment]
@li-boxuan  please assign this to me
